tests/console/curl_https.pm

```diff
@@ -27,7 +27,7 @@ sub run {
     my $max_retries = 7;
     for (1 .. $max_retries) {
         eval {
-            validate_script_output('curl -f -v https://httpbin.org/get 2>&1', sub { m,subjectAltName:[\w\s]+["]?httpbin.org["]? matched, });
+            script_run('curl -f -v https://httpbin.org/get 2>&1 | grep subjectAltName: | grep httpbin.org | grep matched');
```

Comment:
I suggest to use `assert_script_run` (or check the return code)

Reply:
ok, thanks